SimpleJavaProject/src/main/java/alexandr_samohvalov/NumbersMain.java

```diff
@@ -0,0 +1,49 @@
+package alexandr_samohvalov;
+
+import java.util.Random;
+
+public class NumbersMain {
+    //Find the square root of a number
+    public static void ssaww(String[] args) {
+        int a = 49;
+        System.out.println(Math.sqrt(a));
+    }
+
+    // Generate a random number between 0 and 100.
+    public static void maaasfin(String[] args) {
```

Comment:
please review all methods and fix method name and unused parameter indicated previously